src/api.rs

```diff
@@ -3,13 +3,13 @@
 //!
 //! This API is provided as Traits to be implemented:
 //!
-//! - by the low-level layers of the FoxBox, including the adapters;
+//! - by the low-level layers of `FoxBox`, including the adapters;
```

Comment:
New rule of clippy: https://github.com/Manishearth/rust-clippy/wiki#doc_markdown